pubsub/src/main_sub.cpp

```diff
@@ -49,28 +51,28 @@ void signalHandler(int signal) {
 class CustomListener : public UListener {
 
     public:
-        /* in this example the same onReceive callback implementation is used to receive
-         * the three different messages , each message is differntiated by the URI
-         * it is possible to provide a different onReceive callback for each topic */
```

Comment:
https://github.com/eclipse-uprotocol/up-zenoh-example-cpp/pull/8#discussion_r1557253446

@MelamudMichael the comment was removed here. If it needs to be added back in, then it should be done as part of this PR.